### ISSUE TITLE
Nextcloud Upload photos or videos modal not accessible to VoiceOver

### ISSUE THREAD

**Issue by joonshakya** [nextcloud/ios#3069]
### Steps to reproduce

1. Turn on VoiceOver (iOS Screen Reader)
2. Open Nextcloud, click + button in the tab bar, Click "Upload photos or videos"
3. A custom modal will pop up with the gird view of Camera Roll. VoiceOver is not able to select photos or videos from this screen

### Expected behaviour

VoiceOver should be able to select and de-select photos and videos to upload.

### Actual behaviour

VoiceOver doesn't focus on the photo grid area.

### Screenshots

<img width="414" alt="image" src="https://github.com/user-attachments/assets/8e8c3139-facd-4eb2-b52e-98b86dca93d1">

### Logs

```
N/A
```

### Reasoning or why should it be changed/implemented?

It is not accessible for screen readers to choose photos or videos to Nextcloud.

### Environment data

**iOS version:** iOS 18.0

**Nextcloud iOS app version:** Nextcloud Hydropgen for iOS 5.5.5.0 @ 2024

**Server operating system:** Linux 4.18.0-513.9.1.lve.el8.x86_64 x86_64

**Web server:** Apache

**Database:** MySQL 10.6.19

**PHP version:** 8.1.29

**Nextcloud version:** Nextcloud Hub 9 (30.0.0)

**Comment by joonshakya** [comment]
If it shows the native iOS Media Picker then there may not be any issues with VoiceOver compatibility
